source/backend/database/models/usersModel.js

```diff
@@ -12,4 +12,34 @@ async function getByUserId(userId) {
   return result;
 }
 
-module.exports = { getByUserId };
+/**
+ * Searches all completed recipes by userId
+ * @param userId
+ * @returns {Promise<awaited Knex.QueryBuilder<TRecord, TResult>>}
+ */
+async function getCompletedRecipes(userId){
```

Comment:
Since this is querying the `completeRecipes` table, it should be in it's own file maybe a `completeRecipesModel.js` not in the `usersModel` as the first class citizen in this case is not user, but the completeRecipe record.

Reply:
fixed